internal/controller/services/certconfigmapgenerator/certconfigmapgenerator_test.go

```diff
@@ -87,12 +101,7 @@ func TestCertConfigmapGeneratorReconciler(t *testing.T) {
 		})
 
 		g.Expect(err).ShouldNot(HaveOccurred())
-
-		g.Consistently(func() ([]corev1.ConfigMap, error) {
-			return listCABundleConfigMaps(ctx, env.Client())
-		}).Should(
-			BeEmpty(),
-		)
+		g.Consistently(listCABundleConfigMaps(ctx, env.Client())).Should(BeEmpty())
 	})
 
 	t.Run("TrustedCABundle set to Removed", func(t *testing.T) {
```

Comment:
Nil and Removed test cases are already covered at the end, right? So your last push is not needed afaik.

Reply:
fixed